MPU/include/pedalHW.h

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file pedalsHW.h
+ * @author Nick DePatie
+ * @brief Pedal Hardware Abstraction
+ * @date 2022-06-30
+ */
+
+#ifndef PEDALHW_H
+#define PEDALHW_H
+
+#include <nerduino.h>
+
+//Maximum ADC value based on resolution (10 bit resolution for Teensy 4.1)
+#define MAX_ADC_VALUE               1023 //1111111111 in binary
+
+//Timer macros
```

Comment:
One concern I have with the general restructure is how many places parameters are located. This isnt specific to this PR, but thoughts on how we can centralize settings? Or at least group them more closely?

Reply:
We could have one giant file of macros or something, it would probably live in the platformio.ini file like a cmake file's definitions. Would you want to have all the macros just in one file and then separated well with comments?

Reply:
Don't worry for now, but I may want to try it out